src/main/kotlin/ar/edu/unahur/obj2/vendedores/CentrosDeDistribucion.kt

```diff
@@ -0,0 +1,25 @@
+package ar.edu.unahur.obj2.vendedores
+
+import java.lang.Exception
+
+class CentrosDeDistribucion(val ciudad: Ciudad) {
+    val vendedores = mutableListOf<Vendedor>()
+
+    fun agregarVendedor(vendedor: Vendedor) =
+            if(!vendedores.contains(vendedor)){
+                vendedores.add(vendedor)
+            }
+            else{
+                throw Exception ("El vendedor ya está registrado para el centro")
+            }
```

Comment:
Acá sugiero hacerlo al revés.

Si se ejecuta la rama verdadera del `if`, el método explota y nunca llega al `else`. Podés entonces obviarlo, te quedaría así:

```suggestion
            if(vendedores.contains(vendedor)){
                throw Exception ("El vendedor ya está registrado para el centro")
            }
            
            vendedores.add(vendedor)
```